src/bvals/fc/bvals_fc.cpp

```diff
@@ -161,7 +161,7 @@ int FaceCenteredBoundaryVariable::ComputeFluxCorrectionBufferSize(
 //                                                               const NeighborBlock& nb)
 //  \brief Set face-centered boundary buffers for sending to a block on the same level
 
-int FaceCenteredBoundaryVariable::LoadBoundaryBufferSameLevel(Real *buf,
+int FaceCenteredBoundaryVariable::LoadBoundaryBufferSameLevel(ParArray1D<Real> &buf_tmp,
```

Comment:
This is sort of a more general question: Since views have reference semantics, when do we pass them by reference and when by value? I suppose that passing by reference is still best since the metadata isn't copied over.